app/controllers/hub/clients_controller.rb

```diff
@@ -98,7 +98,7 @@ def edit_take_action
         @client,
         current_user,
         tax_return_id: params.dig(:tax_return, :id)&.to_i,
-        current_state: params.dig(:tax_return, :current_state),
+        status: params.dig(:tax_return, :current_state) || params.dig(:tax_return, :status),
```

Comment:
it might be nice if this form consumed the parameter as `updated_status` because that's what it says on the form label

but not a blocker I guess